cypress/e2e/tables-import.cy.js

```diff
@@ -30,3 +32,56 @@ describe('Import csv', () => {
 	})
 
 })
+
+describe('Import csv from Files file action', () => {
+
+	before(function() {
+		cy.createRandomUser().then(user => {
+			localUser = user
+		})
+	})
+
+	beforeEach(function() {
+		cy.login(localUser)
+		cy.visit('apps/files/files')
+		cy.uploadFile('test-import.csv', 'text/csv')
+	})
+
+	it('Import to new table', () => {
+		cy.reload()
+
+		cy.get('[data-cy-files-list-row-actions] .action-item button').click()
+		cy.get('[data-cy-files-list-row-action="import-to-tables"]').click()
+		cy.get('[data-cy="fileActionImportButton"]').click()
+
+		cy.wait(5200)
```

Comment:
`cy.wait` is an anti-pattern as it makes tests flaky or slower then needed.

best here would be to intercept the HTTP request and wait for it to finish.

See https://github.com/nextcloud/tables/blob/f918c273decd57b23988fe99e16258f4c6ae208c/cypress/e2e/view.cy.js#L69-L73 for examples

Reply:
Yep, gotcha. Didn't figure out it was a thing until working on the tests for archive/favorites. Will fix